tests/collections/speechlm2/test_metrics.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from nemo.collections.speechlm2.parts.metrics import BLEU
+
+
+def test_bleu():
+    metric = BLEU(verbose=False)
```

Comment:
Only one metric to test for S2S modules?

Reply:
The other metric, ASRBLEU, is incovenient to test as it requires both a pretrained ASR and an audio file.